test/recipes/30-test_evp.t

```diff
@@ -14,9 +14,11 @@ use OpenSSL::Test qw/:DEFAULT data_file/;
 
 setup("test_evp");
 
-my @files = ( "evpciph.txt", "evpdigest.txt", "evpencod.txt", "evpkdf.txt",
-    "evpmac.txt", "evppbe.txt", "evppkey.txt", "evppkey_ecc.txt",
-    "evpcase.txt" );
+#my @files = ( "evpciph.txt", "evpdigest.txt", "evpencod.txt", "evpkdf.txt",
+#    "evpmac.txt", "evppbe.txt", "evppkey.txt", "evppkey_ecc.txt",
+#    "evpcase.txt" );
+
+my @files = ( "evpmac.txt" );
```

Comment:
I think you didn't mean to do that.  Besides, you can run the test manually, like this:

```
./util/shlib_wrap.sh test/evp_test  test/recipes/30-test_evp_data/evpmac.txt
```

Reply:
What a magic I didn't know before!

Reply:
Reverted, thanks for providing the 'incantation' ;-)